wishlist.css

```diff
@@ -0,0 +1,8 @@
+.wishlist-card-container{
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-items: center;
+    min-height: 79vh;
+    gap:2rem;
```

Comment:
Cards are looking great. HTML codes are semantically correct and clean. Great work.